xgcm/grid.py

```diff
@@ -1359,6 +1361,26 @@ def iterate_axis_combinations(items):
         # return the product of the metrics
         return functools.reduce(operator.mul, metric_vars, 1)
 
+    def interp_like(self, da_source, metric):
+        for axname, axis in self.axes.items():
```

Comment:
This function is part of the public API and should get a [docstring](https://www.python.org/dev/peps/pep-0257/)